packages/react-runtime/src/useRouteRender.ts

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+interface RouteRender {
+  renderTime?: number;
+}
+
+export function useRouteRender(): RouteRender {
+  const [routeRender, setRouteRender] = React.useState<RouteRender>({});
```

Comment:
这样吧，可以更好的区分*已渲染*和*未渲染*

```suggestion
  const [routeRender, setRouteRender] = React.useState<RouteRender | null>(null);
```